packages/manager/modules/pci/src/components/project/users/translations/Messages_fr_FR.json

```diff
@@ -34,7 +34,8 @@
   "pci_projects_project_users_add_label": "Ajouter un utilisateur",
   "pci_projects_project_users_matrix_show": "Afficher la matrice des rôles",
   "pci_projects_project_users_matrix_hide": "Masquer la matrice des rôles",
-  "pci_projects_project_users_generate_s3_credentials": "Générer les certificats S3",
+  "pci_projects_project_users_generate_s3_credentials": "Générer les credentials S3",
+  "pci_projects_project_users_high_perf_info": "Les rôles de l'offre High Performance object storage sont à définir dans votre offre Object storage, onglet \"Utilisateurs S3\".",
```

Comment:
```suggestion
  "pci_projects_project_users_high_perf_info": "Les rôles de l'offre High Performance Object storage sont à définir dans votre offre Object storage, onglet \"Utilisateurs S3\".",
```

Reply:
This is actually discussed in the comments section of TRANS-40700

Reply:
Close this as it has been decided it will be handled later